float64[] args
---
int32 res